x-pack/plugins/security_solution/cypress/integration/detection_rules/prebuilt_rules.spec.ts

```diff
@@ -36,50 +36,64 @@ import { loginAndWaitForPageWithoutDateRange } from '../../tasks/login';
 
 import { ALERTS_URL } from '../../urls/navigation';
 
-import { totalNumberOfPrebuiltRules } from '../../objects/rule';
```

Comment:
If we still want a sanity check, we could do a `>=` comparison on this.
You can never use fleet to _remove_ rules.

Reply:
Ah, I will add this back and remove the intercept as this would be a stronger way to test.